tools/run-materialize.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+dl_materialized () {
+	INSTALL_DIR=$1
+	git -C $INSTALL_DIR clone git@github.com:MaterializeInc/materialize.git
+	cd $INSTALL_DIR"/materialize"
+}
+
+edit_rp_service () {
+	SERVICE_FILE_PY=$1"/materialize/misc/python/materialize/mzcompose/services.py"
+
+	TARGET_VERSION=$2
+	echo "Using RP version: $TARGET_VERSION"
+
+	DEFAULT_RP_VERSION="v21.11.13"
+	sed -i'' "s@$DEFAULT_RP_VERSION@$TARGET_VERSION@" $SERVICE_FILE_PY
+
+	STABLE=$3
+	if [ "$STABLE" = "unstable" ]
+	then
+		echo "Using unstable repo"
+		sed -i'' "s@vectorized/redpanda@vectorized/redpanda-unstable@" $SERVICE_FILE_PY
+	fi
+}
+
+run_test_suite () {
+	cd $1"/materialize/test/testdrive"
+	./mzcompose run default --redpanda
+
+	./mzcompose down -v
+}
+
+if [ $# -lt 2 ]
+then
+    echo "Usage: $0 <install dir> <rp version> [unstable]"
+    echo "Example: $0 /home/user/Documents v22.1.3-rc1 unstable"
+else
+	INSTALL_DIR=$1
+	RP_VERSION=$2
+
+	# Set stable as default value using
+	# bash param expansion
+	STABLE=${3:-"stable"}
```

Comment:
betas (v21) appear to be in stable, whilst rcs (v22+) are in unstable.

I think this can be automatic, based on whether the version contains a `-`, and is v22:
```bash
    if [[ "$TARGET_VERSION" == *"-"* && "$TARGET_VERSION" != "v21"* ]]; then
        STABLE=${3:-"unstable"}
    else
        STABLE=${3:-"stable"}
    fi
```

Reply:
done.